### PR TITLE
MLE-15255 : Run the autogeneration of new Optic Code for Node Client

### PR DESCRIPTION
**Pull request by anu3990**

### CODE REVIEW

File: lib/plan-builder-generated.js [hunk 1]
```diff
@@ -1194,8 +1194,7 @@ pointLongitude(...args) {
     * @returns { CtsPolygon }
     */
 polygon(...args) {
-    // Adding types.XsString manually since cts.polygon now accepts string input (MarkLogic 11.1.0 or higher needed).
-    const paramdef = ['vertices', [types.CtsPoint, PlanColumn, PlanParam, types.XsString], false, true];
+    const paramdef = ['vertices', [types.CtsPoint, PlanColumn, PlanParam], false, true];
     const checkedArgs = bldrbase.makeSingleArgs('cts.polygon', 1, paramdef, args);
```
Comment (anu3990) [inline lib/plan-builder-generated.js:1198]: Hi Aaron and Ling, the changes on lines 1197 and 1161 (cts.point and cts.polygon) came in as part of auto-generation but the tests are now failing. Please let me know if we can disregard the incoming changes from the auto-generation and revert it back.

Comment (rjrudin) [reply lib/plan-builder-generated.js:1198]: Should definitely revert those changes. This is a good example of a comment added to a generated file helping us understand why it should be reverted.

File: lib/plan-builder-generated.js [hunk 2]
```diff
@@ -8540,7 +8760,7 @@ fromDocDescriptors(...args) {
 
     }
 /**
-    * This function matches and returns the uri, content, and score for documents. Provides a client interface to a server function. See {@link http://docs.marklogic.com/op.fromDocUris|op.fromDocUris}
+    * This function matches and returns the uri for documents. Provides a client interface to a server function. See {@link http://docs.marklogic.com/op.fromDocUris|op.fromDocUris}
```
Comment (rjrudin) [inline lib/plan-builder-generated.js:8763]: This is fine for now, but in the future, any uses of "uri" (unless referring to a column name) should be changed in the hint/codegen file to be "URI".

**Review by rjrudin — COMMENTED** [review]
Looks good, but definitely revert those changes to point/polygon, as that's undoing important additions for 11.1.0.

**Comment by anu3990** [comment]
Moving ahead with https://github.com/marklogic/node-client-api/pull/880

**Pull request closed without merging**